R/create_single_dose_dataset.R

```diff
@@ -411,15 +416,38 @@ create_single_dose_dataset <- function(dataset,
   dataset_part_2 <- dataset %>%
     filter(!!dose_freq != "ONCE")
 
-  dataset_part_2 <- dataset_part_2 %>%
-    left_join(lookup, by = as.character(quo_get_expr(dose_freq))) %>%
-    mutate(dose_periods = case_when(
+  if (quo_is_null(start_datetime)) {
+    min_hour_cases <- rlang::exprs(FALSE ~ 0)
```

Comment:
Do we have guidance on when to use `pkg_name::` inside the function body? Should `exprs` just be included in the:
 `#' @importFrom rlang` portion of `R/admiral-package.R` ?

Reply:
We do not have a strict guidance.

I have added `exprs()` to `admiral-package.R`.